src/ApiManagement/ApiManagement.ServiceManagement/Commands/GetAzureApiManagementPolicy.cs

```diff
@@ -43,8 +43,9 @@ public class GetAzureApiManagementPolicy : AzureApiManagementCmdletBase
         [Parameter(
             ValueFromPipelineByPropertyName = true,
             Mandatory = false,
-            HelpMessage = "Format of the policy. Default value is ‘application/vnd.ms-azure-apim.policy+xml’." +
+            HelpMessage = "Format of the policy. Default value is ‘xml’." +
                           " This parameter is optional.")]
+        [PSArgumentCompleter("xml", "rawxml")]
```

Comment:
@solankisamir for the values in the `PSArgumentCompleter`, can you use PascalCase to name the values?

This comment applies to all instances of `PSArgumentCompleter` in this PR